packages/web-components/src/assets/chevron-icon.svg

```diff
@@ -1,3 +1,3 @@
-<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The icon doesn't scale when you change the browser font size e.g. see compared to the chevron on the bottom left that does:

<img src="https://user-images.githubusercontent.com/111882109/221906498-0d2b8ddb-bc78-45cc-9310-bb2cf634befa.png" width="300">

May need to give it a height / width in `rem` in the CSS?